GameServerApp/Program.cs

```diff
@@ -17,13 +17,17 @@ public class ArgsOptions
         [Option("config", DefaultValue = "Settings/GameInfo.json")]
         public string ConfigPath { get; set; }
 
+        [Option("configJSON", DefaultValue = "")]
```

Comment:
I'd prefer `config-json` as the param name here.

In my opinion, it'd be better if the json file path was used instead. This allows for easier debugging on later stages when complicated configurations are passed for the server, as a new json is generated for each started game. Another benefit is that a history/log of previous configurations is automatically created.